frontend/package-lock.json

```diff
@@ -12847,9 +12847,9 @@
       "dev": true
     },
     "node_modules/@types/node": {
-      "version": "15.12.4",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-15.12.4.tgz",
-      "integrity": "sha512-zrNj1+yqYF4WskCMOHwN+w9iuD12+dGm0rQ35HLl9/Ouuq52cEtd0CH9qMgrdNmi5ejC1/V7vKEXYubB+65DkA=="
+      "version": "16.18.11",
```

Comment:
Why did the `frontend/package-lock.json` changed? Please reset it

Reply:
Can you please reset changes made to that file? This commend was not addressed (or it was changed again?)

Reply:
This was still not addressed!